packages/ui/src/views/repo/pull-request/details/components/conversation/sections/pull-request-merge-section.tsx

```diff
@@ -49,145 +91,103 @@ const PullRequestMergeSection = ({
-                      >
-                        {/* {getString('commandLine')} */}
-                        command line
-                      </span>
-                      to resolve conflicts
-                      {/* {getString('pr.useCmdLineToResolveConflicts')} */}
-                    </p>
-                  </Layout.Vertical>
-                </div>
-              ) : (
-                ''
-              )
-            }
-          />
-          {!mergeable && !unchecked && (
-            <span className="px-2 py-1.5 text-14 text-foreground-2 transition-colors duration-200 group-hover:text-foreground-1">
-              Show more
-            </span>
-          )}
-        </Accordion.Trigger>
-        {!mergeable && !unchecked && (
-          <Accordion.Content>
-            <div className="ml-6">
-              {showCommandLineInfo && (
-                <div className="mb-2 rounded-md border border-border p-1 px-4 py-2">
-                  <h3 className="text-14 text-foreground-1">Resolve conflicts via command line</h3>
-                  <p className="pb-3 pt-1 text-14 text-foreground-4">
-                    If the conflicts on this branch are too complex to resolve in the web editor, you can check it out
-                    via command line to resolve the conflicts
+    <Accordion.Item value={ACCORDION_VALUE} isLast>
+      <Accordion.Trigger
+        className="py-3 text-left [&>svg]:-rotate-0 [&>svg]:data-[state=open]:-rotate-180"
+        chevronClassName="text-icons-3 self-start mt-1"
+        hideChevron={mergeable || unchecked}
+      >
+        <StackedList.Field
+          className="flex gap-y-1"
+          title={
+            <LineTitle
+              textClassName={isConflicted ? 'text-foreground-danger' : ''}
+              text={
+                unchecked
+                  ? 'Merge check in progress...'
+                  : !mergeable
+                    ? 'Conflicts found in this branch'
+                    : `This branch has no conflicts with ${pullReqMetadata?.target_branch} branch`
+              }
+              icon={
+                unchecked ? (
+                  <Icon name="clock" className="text-icons-alert" />
+                ) : (
+                  <Icon
+                    className={mergeable ? 'text-icons-success' : 'text-icons-danger'}
+                    name={mergeable ? 'success' : 'triangle-warning'}
+                  />
+                )
+              }
+            />
+          }
+          description={
+            <>
+              {unchecked && <LineDescription text={'Checking for ability to merge automatically...'} />}
+              {isConflicted && (
+                <Layout.Vertical className="ml-6">
+                  <p className="text-14 font-normal text-foreground-4">
+                    Use the&nbsp;
+                    <Button variant="link_accent" size="text" onClick={handleCommandLineClick} asChild>
+                      <span>command line</span>
```

Comment:
Curious why we'd need the empty `span`?
```suggestion
                      command line
```

With the `asChild`, it would assign a click handler to a `span` which would be terrible for accessibility.

Reply:
Because we are using the Button with the asChild prop, and we need to pass an Element so that all the button's properties are applied to it.

Reply:
For accessibility reasons, we should _never_ attach click handlers to things that aren't buttons or anchors. Using `asChild` here makes a `span` clickable, with no indication to assistive technologies that it can be clicked. It won't be read out to screen readers and cannot be focused by keyboard users. Below is what the screen reader would see for this kind of element; it is seen as a "generic" element.  
![Screenshot 2025-03-24 at 10 56 23](https://github.com/user-attachments/assets/f8daa075-a677-424a-90ba-f1659bd3cd94)

Reply:
Due to the fact that the current text is already rendered in a button, I can't use a button again, so I'm using a span. However, for screen readers, I've added `role`, `tabIndex`, `aria-label`, and `onKeyDown`

Reply:
Any reason to not just use without `asChild` and render a normal button?